docs/usage.rst

```diff
@@ -200,9 +202,13 @@ In this setup, all the user has to do is executing ``conda devenv``:
     $ cd ~/projects/web-ui
     $ conda devenv
 
-This will create a ``conda`` environment named ``web-ui`` containing all the dependencies and environment variables
+This will create a ``conda`` environment named ``web-ui`` merging all the dependencies and environment variables
 defined in both files.
 
+However, the same environment variable defined as a single string (like DB_LOCATION above) in both files will raise an error
+unless it is not allowed to 'pass through' using the ``not is_included`` expression above as an example.
+In other words, an 'overwrite' situation is not allowed between files.
```

Comment:
```suggestion
However, the same environment variable defined as a single string (like ``DB_LOCATION`` above) in both files will raise an error
unless it is not allowed to 'pass through' using the ``# [not is_included]`` selector above as an example.
In other words, an 'overwrite' situation is not allowed between files.
```

The syntax `# [some_bool_expr]` have been copied from [conda build and they call it preprocessing selectors](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors) I think we should you the same term. Actually [our docs describe the feature in jinja section](https://conda-devenv.readthedocs.io/en/latest/usage.html#jinja2) but fails to point the origin and I think we should use the same name and add an reference to the conda-build.